### ISSUE TITLE
 A little question about PocketMiner training set

### ISSUE THREAD

**Issue by HeJunhong1107** [Mickdub/practice_tagging#1]
First of all， I am very sorry for raising this issue under inappropriate repository, because the PocketMinder repository seems to be unable to raise issues, and I did not find your email address.

In the training dataset of PocketMiner, I noticed that because of the uncertainty of MD, the proteins structure will have different labels. But there were 10 MDs in parallel for the same input protein. This means that in the first 40ns MD window, the initial structures are the same, but the labels are indeed different. This is unreasonable for model training，may I ask how to select the protein structure for model training in the first 40ns window? Is it via Fast instead of the first structure?
